Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp

```diff
@@ -299,12 +299,14 @@ void GeneralWidget::AddDescriptions()
                  "cost of high GPU requirements. Only recommended for high-end systems.");
   static const char* TR_SHADER_COMPILE_ASYNC_UBER_DESCRIPTION =
       QT_TR_NOOP("Ubershaders will be used to prevent stuttering during shader compilation, but "
-                 "specialized shaders will be used when they will not cause stuttering.");
+                 "specialized shaders will be used when they will not cause stuttering. Balances "
+                 "GPU strain and stutter reduction.");
   static const char* TR_SHADER_COMPILE_ASYNC_SKIP_DESCRIPTION =
       QT_TR_NOOP("Instead of using ubershaders during shader compilation, objects which use these "
                  "shaders will be not be rendered. This can further reduce stuttering and "
                  "performance requirements, compared to ubershaders, at the cost of introducing "
-                 "visual glitches and broken effects. Not recommended.");
+                 "visual glitches and broken effects. Not recommended unless the other options are "
```

Comment:
Me and Stenzek talked about updating this description, but then Stenzek went on holiday and everyone forgot. We liked this for Skip Ubershaders:

> Prevents shader compilation by not rendering waiting objects. Can work in scenarios where Ubershaders doesn't, at the cost of introducing visual glitches and broken effects. Not recommended, only use if the other options give poor results on your system. 

Would you mind implementing this?

Reply:
Fine with me, that last sentence is important.